ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/EeaSendTransactionJsonParameters.java

```diff
@@ -49,7 +49,7 @@ public EeaSendTransactionJsonParameters(
       @JsonProperty("from") final String sender,
       @JsonProperty("privateFrom") final String privateFrom,
       @JsonProperty("restriction") final String restriction) {
-    validatePrefix(sender);
+    validateNotEmpty(sender);
```

Comment:
Is there a benefit to allow any String here? Before they were checking whether the String starts with a 0x, which does not guarantee that this is a valid hex string. Should we check that only valid addresses are used, e.g. the first two characters can be 0x, but don't have to be. There have to be x number of vaild characters [0, 10] [a, f] [A, F]?
If this is not a valid address then we should fail as soon as possible with a meaningful error message?

Reply:
The benefit is that we don't do the validation it will be done in Web3j or Besu and that we don't have to maintain this validation. Also like you said the 0x check wasn't checking that it was actually hexadecimal.